packages/drizzle/src/postgres/init.ts

```diff
@@ -74,6 +74,7 @@ export const init: Init = async function init(this: BasePostgresAdapter) {
         disableNotNull: !!collection.versions?.drafts,
         disableUnique: true,
         fields: versionFields,
+        joins: collection.joins,
```

Comment:
Is this being used any longer since you deleted the case 'join' field in traverseFields?